### PR TITLE
Finnish fixes 10

### PR DESCRIPTION
**Pull request by fergusq**

### CODE REVIEW

File: mem-12-q.xml [hunk 1]
```diff
@@ -11698,6 +11700,7 @@ There is no one way to say "firefighter". Some options are {qul SuvwI':n} "fire
       <column name="part_of_speech">n:being,deriv</column>
       <column name="definition">governor</column>
       <column name="definition_de">Gouverneur, Regierungsinhaber</column>
+
```
Comment (De7vID) [inline mem-12-q.xml:11703]: Remove this newline.

File: mem-07-l.xml [hunk 2]
```diff
@@ -8190,7 +8190,7 @@ Marc Okrand has said that he prefers to call the puns or "Easter Eggs" in the Kl
       <column name="definition_zh_HK">災難</column>
       <column name="definition_pt">catástrofe</column>
       <column name="definition_fi">katastrofi</column>
-      <column name="synonyms"></column>
+      <column name="synonyms">{Qugh:n:1h}</column>
```
Comment (De7vID) [inline mem-07-l.xml:8193]: This is already linked in the `notes` section. (In general, when something is in the notes section, it's excluded from fields like `synonyms` and `see_also`, but probably this hasn't been done consistently.)

Also, I would prefer if changes to translations to be isolated from other changes, so changes like these probably belong in a separate PR.

File: mem-12-q.xml [hunk 3]
```diff
@@ -11671,25 +11671,27 @@ There is no one way to say "firefighter". Some options are {qul SuvwI':n} "fire
       <column name="notes_ru"></column>
       <column name="notes_zh_HK"></column>
       <column name="notes_pt"></column>
-      <column name="notes_fi"></column>
+      <column name="notes_fi">Englanninkielinen määritelmä "government" tarkoittaa yleensä hallitusta, mutta yhdysvaltojen englannissa laajemmin valtiota kokonaisuudessa. Esimerkki varmistaa, että klingonin sanassa on kyse jälkimmäisestä merkityksestä.</column>
       <column name="hidden_notes"></column>
       <column name="components"></column>
-      <column name="examples"></column>
+      <column name="examples">
+▶ {qum DuSaQ:n} "public school"</column>
```
Comment (De7vID) [inline mem-12-q.xml:11678]: When an example has a linked entry, it's better not to provide another translation here as the two can get out of sync if one of them changes. (There are actually probably a couple of cases like this in the database.) 

I think the right thing to do is to copy the translation from the entry in code and automatically display it. (One day I will redesign the database and rewrite the code to make this easier.)

**Review by De7vID — COMMENTED** [review]
Thanks for the fixes.

File: mem-07-l.xml (hunk 2 above)
Comment (De7vID) [reply mem-07-l.xml:8193]: Is there a reason you want this in `synonyms`, e.g., you're using that field to build a graph of relationships between entries or something?

File: mem-09-n.xml [hunk 4]
```diff
@@ -880,7 +880,7 @@
       <column name="definition_fi">hedelmä, vihannes</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
-      <column name="see_also">{baQ:v:2}, {DeH:v}, {Du' naH:n}, {naH tlhab:n}, {ghub:n}, {qurgh:n}, {'oQqar:n}, {'aquta':n}, {yub:n}, {tIr:n}, {yob:v}, {wIj:v}, {yotlh:n}</column>
+      <column name="see_also">{baQ:v:2}, {DeH:v}, {Du' naH:n}, {naH tlhab:n}, {ghub:n}, {qurgh:n}, {'oQqar:n}, {'aquta':n}, {yub:n}, {tIr:n}, {yob:v}, {wIj:v}, {yotlh:n}, {peb'ot:n}</column>
```
Comment (De7vID) [inline mem-09-n.xml:883]: (This is probably a case where I should split off "examples of the thing" into a list under `notes`, as it's getting a bit long. Just noting to myself to do this later.)

File: mem-03-D.xml [hunk 5]
```diff
@@ -10742,9 +10742,9 @@ The logo of Star Alliance, the largest airline alliance on Earth, is often paint
       <column name="definition_zh_HK">中午</column>
       <column name="definition_pt">meio dia</column>
       <column name="definition_fi">keskipäivä</column>
-      <column name="synonyms"></column>
+      <column name="synonyms">{pemjep:n}</column>
```
Comment (De7vID) [inline mem-03-D.xml:10745]: So there was actually a reason why {DungluQ} and {pemjep} were not already synonyms. People weren't sure they were quite the same thing, as we don't know exactly how time-keeping works on {Qo'noS}. It's also not clear whether one or both of these refer to just a point in time or a period of time (as "midday" can sometimes refer to a period of time around noon in English). But since these are both already marked as antonyms of {ramjep}, making them synonyms seems like the right thing.

File: mem-07-l.xml (hunk 2 above)
Comment (fergusq) [reply mem-07-l.xml:8193]: Yes, I was thinking of using boQwI' as a basis for a thesaurus with different world relations. My first application would be evaluating word vectors I have created. Another idea I have is to create a tool that could suggest related words that rhyme, to assist poem writing.

The problem with the notes section is that it is not machine-readable. Perhaps it could contain tags invisible to the boQwI' user like:

```xml
<column name="notes">This word and {Qugh:n:1h:synonym} are mostly interchangeable. To refer to something that in all likelihood one can't recover from (like the loss of {pIraqSIS:n}), {lot:n:nolink} is better.[2]</column>
```
